scrapli/transport/plugins/telnet/transport.py

```diff
@@ -218,7 +226,7 @@ def _read(self, n: int = 65535) -> None:
                     self._raw_buf += buf
                 else:
                     self._cooked_buf += buf
-            except Exception as exc:
+            except EOFError as exc:
```

Comment:
Minor thing, but I change this to be less broad. Unrelated to the rest of the changes.